src/linux/vnode.c

```diff
@@ -170,14 +165,17 @@ delete_watch(struct filter *filt, struct knote *kn)
 int
 evfilt_vnode_copyout(struct kevent *dst, struct knote *src, void *ptr UNUSED)
 {
-    struct inotify_event evt;
+    char buf[sizeof(struct inotify_event) + NAME_MAX + 1];
+    struct inotify_event *p_evt;
     struct stat sb;
 
-    if (get_one_event(&evt, src->kdata.kn_inotifyfd) < 0)
+    if (get_one_event(buf, sizeof(buf), src->kdata.kn_inotifyfd) < 0)
         return (-1);
 
-    dbg_printf("inotify event: %s", inotify_event_dump(&evt));
-    if (evt.mask & IN_IGNORED) {
+    p_evt = (struct inotify_event *)buf;
```

Comment:
just keep is as evt.  I looked through the rest of the code and they don't use that convention.